src/content/pt/2019/accessibility.md

```diff
@@ -0,0 +1,317 @@
+
+Para garantir que o seu site seja legível nessas condições, é fundamental que o texto tenha contraste de cor suficiente com o fundo. Também é importante considerar quais contrastes serão mostrados quando as cores forem convertidas para tons de cinza.
+
+
+{{ figure_markup(
+  image="example-of-good-and-bad-color-contrast-lookzook.svg",
+  caption="Exemplo de como é o texto com contraste de cor insuficiente. Cortesia de LookZook",
+  description="Quatro caixas coloridas de tons de marrom e cinza com texto branco sobreposto dentro, criando duas colunas. A coluna da esquerda diz Muito levemente colorido e tem a cor de fundo marrom escrita como <code> # FCA469 </code>. A coluna da direita diz Recomendado e a cor de fundo marrom é escrita como <code> # BD5B0E </code>. A caixa superior em cada coluna tem um fundo marrom com texto branco <code> #FFFFFF </code> e a caixa inferior tem um fundo cinza com branco text <code> #FFFFFF </code>. Os equivalentes em tons de cinza são <code> # B8B8B8 </code> e <code> # 707070 </code> respectivamente. Cortesia de LookZook",
+  width=568,
+  height=300
+  )
+}}
+
+Apenas 22,04% dos sites deram contraste de cor suficiente a todo o texto. Ou em outras palavras: 4 em cada 5 sites têm texto que se mistura facilmente com o fundo, tornando-o ilegível.
+
+<p class = "note"> Observe que não foi possível analisar nenhum texto dentro das imagens, portanto, nossa métrica relatada é um limite superior do número total de sites que passaram no teste de contraste de cor. </p>
+
+### Ampliando (zoom) e dimensionando páginas
+
+Usando um [tamanho de fonte legível](https://accessibleweb.com/question-answer/minimum-font-size/) e [tamanho alvo](https://www.w3.org/WAI/WCAG21/quickref/#target-size) ajuda os usuários a ler e interagir com seu site. Mas mesmo os sites que seguem perfeitamente todas essas diretrizes não podem atender às necessidades específicas de cada visitante. É por isso que recursos do dispositivo, como a pinça para aplicar zoom e dimensionamento, são tão importantes: eles permitem que os usuários ajustem suas páginas para que suas necessidades sejam atendidas. Ou, no caso de sites particularmente inacessíveis que usam fontes e botões minúsculos, dá aos usuários a chance de usar o site.
+
+
+Existem raros casos em que a desativação do escalonamento é aceitável, como quando a página em questão é um jogo baseado na web que usa controles de toque. Se deixado ativado neste caso, os telefones dos jogadores irão aumentar e diminuir o zoom cada vez que o jogador tocar duas vezes no jogo, ironicamente tornando-o inacessível.
+
+Por causa disso, os desenvolvedores têm a capacidade de desativar esse recurso definindo uma das duas propriedades a seguir na [meta tag viewport](https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag):
```

Comment:
Can you check if Portuguese (either European or Brazilian) exist for all resources and use them if they do? Quite a few references to MDN in this chapter so only pointing out this first one but can you have a look at the rest?

```suggestion
Por causa disso, os desenvolvedores têm a capacidade de desativar esse recurso definindo uma das duas propriedades a seguir na [meta tag viewport](https://developer.mozilla.org/pt-BR/docs/Mozilla/Mobile/Viewport_meta_tag):
```